datasafe-rest-impl/src/test/java/de/adorsys/datasafe/rest/impl/controller/UserControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package de.adorsys.datasafe.rest.impl.controller;
+
+import de.adorsys.datasafe.business.impl.service.DefaultDatasafeServices;
+import de.adorsys.datasafe.directory.impl.profile.operations.DFSBasedProfileStorageImpl;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+class UserControllerTest extends BaseTokenDatasafeEndpointTest {
+
+    @MockBean
+    DefaultDatasafeServices dataSafeService;
+
+    @MockBean
+    DFSBasedProfileStorageImpl userProfile;
+
+    @BeforeEach
+    public void setup() {
+        when(dataSafeService.userProfile()).thenReturn(userProfile);
+        super.setup();
```

Comment:
Not necessary, Jupiter calls it automatically 
`super.setup()`

Reply:
I don't think so. I tried and it doesn't work

Reply:
I don't think so. I tried and and it doesn't work

Reply:
Simply do not override parent method - rename setup into init

Reply:
ok, thanks